pysparkling/sql/expressions/operators.py

```diff
@@ -1,7 +1,8 @@
 from pysparkling import Row
```

Comment:
This is a bad import and the reason why you had to force import order in one of the `__init__` files.

Overall, for Python modules that are not scripts, all these absolute imports should be replaced by relative imports. But this is can be done later. Not for this PR.